packages/nimble-components/src/table-column/number-text/index.ts

```diff
@@ -46,13 +50,21 @@ export class TableColumnNumberText extends TableColumnTextBase {
     @attr({ attribute: 'decimal-digits', converter: nullableNumberConverter })
     public decimalDigits?: number;
 
+    private readonly langSubscriber: DesignTokenSubscriber<typeof lang> = {
+        handleChange: () => {
+            this.updateColumnConfig();
+        }
+    };
+
     public override connectedCallback(): void {
         super.connectedCallback();
+        lang.subscribe(this.langSubscriber, this);
         this.updateColumnConfig();
     }
 
     public override get validity(): TableColumnValidity {
         return this.validator.getValidity();
+        lang.unsubscribe(this.langSubscriber, this);
```

Comment:
Is this a mistake?

Reply:
Yes...this should be in the `disconnectedCallback` here. I'm not sure how that happened. 🤦‍♂️